mava/utils/environments/smac_utils.py

```diff
@@ -39,13 +39,14 @@ def make_environment(
         concat_agent_id: bool = True,
         evaluation: bool = False,
         random_seed: Optional[int] = None,
+        death_masking: bool = False,
```

Comment:
Please add docstrings here.

Reply:
Thanks @KaleabTessera for the review, @RuanJohn already made benchmark for that [here](https://www.notion.so/instadeep/Experiments-0741b23a72c740c689e1c09753d5217c?p=7e6083a3df1e4aed86427fd89b868136&pm=s). Also I will add the docstring!
I think the issue of whether agents get reward or not while they are dead is handled automatically in smac env but we can double check it!